docs/global-settings.md

```diff
@@ -81,3 +81,29 @@ contentful:
 layout: posts.html
 ---
 ```
+
+### `common` *(optional)*
+
+The results of queries placed in this property will be made available in all templates
+
+For example, find your five latest entries so you can include them in various templates:
+
+```javascript
+{
+  // ...
+  "common": {
+    "latest": {
+      "content_type": "post",
+      "limit": 5,
+      "order": "sys.createdAt"
+    }
+  },
+  // ...
+}
+```
+
+In templates you can then access `common.latest` to get the raw results of the above query:
+
+```
+The first latest post's title: {{ common.latest.items.0.fields.title }}
```

Comment:
not in handlebars land 👨 !

i'll make a note in the example that it's a handlebars template.